### ISSUE TITLE
Set correct author when uploading to Readwise

### ISSUE THREAD

**Issue by ituri** [tibobrc/Blinkist2Readwise#1]
First of all, thanks for your useful script! 

Right now when I import my highlights to Readwise, the author is defined as "Blinkist". Would it be possible to use the real author's name? Instead, you could tag the book as "blinkist".

**Comment by kahyalar** [comment]
I agree with you. But for now Blinkist only provides the highlight and title of the book in the "Highlights" page. Maybe in future, they will change the UI.
